### ISSUE TITLE
Support datasets in coordinates other than WGS84

### ISSUE THREAD

**Issue by kwilcox** [sci-wms/sci-wms#41]
UGridDataset's `getMap` implementation always transforms the passed in BBOX and CRS parameters into WGS84 before querying the dataset.  We should try to pull a valid CRS from a grid_mapping attribute on the data variable and fall back to WGS84 if it doesn't exist or is invalid.  This will allow us to support datasets using any projection proj4 supports.

**Comment by rsignell-usgs** [comment]
@kwilcox , isn't this fixed?

**Comment by kwilcox** [comment]
@rsignell-usgs No, it still assumes a dataset is stored in lat/lon coordinates. The work would be to pull a valid pyproj Proj object from a grid's `grid_mapping` metadata variable instead of assuming WGS84.
